### ISSUE TITLE
508-defect-3: "Claimant contact information" should programmatically describe relevant fields

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#43346]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

"Claimant contact information" is starting a new section within the Step 5 of 6: Additional information page. If we are to assume that the goal of having "Claimant contact information" is to inform the user that the following fields are the claimants, it should be one of the following:
- be programmatically linked to each of the fields labels
- be a part of each of the field labels
- take programmatic focus upon landing on the page

Only some of the prior conditions are partially met as it is currently a `legend` within a `fieldset` that wraps around the address and phone number fields. This results in the following behavior:
- Address fields, as they belong to the fieldset "Address", will not have "Claimant contact information" appended to their fields (nested fieldset issues 😢 ) ❌ 
- Email and phone number will have "Claimant contact information" appended to their fields ✔️ 

## Acceptance Criteria

TBD

- [ ] Screen reader users who navigate using the `tab` key can determine that each of the fields are relevant to the claimant

## Steps to Recreate

1. [Go to the claimant address and information page ](https://www.va.gov/burials-and-memorials/application/530/claimant-contact-information)
2. Turn on voiceover on macbook
3. Confirm that the address field does not announce "claimant contact information"

## Proposed Solution (if known)

Ideally, we could have unique headings and legends for each page following a one thing per page pattern (instead of sending focus to a relatively generic step).

Realistically in the short term, the following bandaid fix could resolve the issue:
- Rename the legend to "Claimant address" instead of "Address"
- Rename the fields to "Claimant's email address" and "Claimant's phone number"

If it's important to note that these fields are being used to contact the claimant; we should retain that language or make it more clear.

## Screenshots or Trace Logs
<img width="815" alt="Screen Shot 2022-06-24 at 12 12 16 PM" src="https://user-images.githubusercontent.com/14154792/175575402-740b1382-d159-4e54-b775-0df1ac16f9c6.png">

**Comment by fiorella-io** [comment]
In the Burial redesigns we've fixed this issue. The claimant's address information appears on one page, and their contact information on another.

Designs:
- [Claimant's address](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/GozZRAv)
- [Claimant's contact information](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/Oe2vPGw)

**Comment by fiorella-io** [comment]
In the new Burial form, we've fixed this issue. The claimant's address appears on one page, and their contact information on another.

Designs:
- [Claimant's address](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/GozZRAv)
- [Claimant's contact information](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/Oe2vPGw)

**Comment by steele-lm** [comment]
@Midge-dev or @DLarson-Oddball Can you confirm the screenreader issues Josh called out in this old ticket have been addressed with the form redesign? If everything looks good, I'll close out this ticket. Thanks!
